frontend/src/components/program/picasso/PicassoEntry.vue

```diff
@@ -174,6 +179,23 @@ export default {
     isLongDuration() {
       return this.scrollHeight <= this.clientHeight
     },
+    durationText() {
+      const start = this.$date(this.scheduleEntry.startTimestamp).utc(true)
```

Comment:
So far we have always used `this.$date.utc(this.scheduleEntry.startTimestamp)`.

The result is the same, but for sake of consistency it would be nice to stick with one way.

Reply:
Unfortunately, your suggestion does not lead to the same result:
![image](https://github.com/ecamp/ecamp3/assets/470237/da969d30-098e-4ddb-8b6c-0b4b391e9247)

Reply:
Ah, sorry, wrongly assumed that .utc(true) is a shorthand for dayjs.utc(...) but seems the result is different.

Although the result seems to be the same, it would be better and more explicit to use `timestampToUtcString` for the conversion between VCalendar timestamps (which are in user's local timezone) and UTC-Datetime-Strings (which we use in DB and to display the times).

```javascript
durationText() {
      const start = timestampToUtcString(this.scheduleEntry.startTimestamp)
      const end = timestampToUtcString(this.scheduleEntry.endTimestamp)
      if (this.dateShort(start) === this.dateShort(end)) {
```